gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/testmanager/CompareResults.java

```diff
@@ -0,0 +1,340 @@
+						.collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue()));
+				System.out.println("     Lookup expected" + expectedOutputMap.get("rcon_VREG3"));
+
+			}
+		} catch (FileNotFoundException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		return expectedOutputMap;
+	}
+
+	/**
+	 * Compare two json objects individual property
+	 * 
+	 * @param simOutputObj
+	 * @param expectedOutputObj
+	 * @param prop
+	 * @return
+	 */
+	public Boolean compareObjectProperties(JsonObject simOutputObj, JsonObject expectedOutputObj, String prop) {
+		Boolean comparison = simOutputObj.get(prop).equals(expectedOutputObj.get(prop));
+		System.out.println("     " + prop +  " : " + simOutputObj.get(prop) + " == " +  expectedOutputObj.get(prop) + " is " + comparison);
+		// Test voltage, i.e. complex number
+		if(prop.startsWith("voltage_") || prop.startsWith("power_in_") ){
+			Complex c1 = cf.parse(simOutputObj.get(prop).getAsString());
+			Complex c2 = cf.parse(expectedOutputObj.get(prop).getAsString());
+			System.out.println("              Complex" + c1 + c2 + equals(c1,c2));
+			comparison = equals(c1,c2);
+		}
+		
+		return comparison;
+	}
+	
+	public void loadSimResult_old(String sim_output, SimulationOutput simOutProperties, String expected_output) {
+		JsonParser parser = new JsonParser();
+		try {
+			BufferedReader br = new BufferedReader(new FileReader(sim_output));
+			JsonElement elm = parser.parse(br);
+			if(elm.isJsonObject()){
+			    JsonObject jsonObject = elm.getAsJsonObject();
+
+			    JsonObject output = jsonObject.get("output").getAsJsonObject();
+
+			    JsonObject f2 = output.get("ieee8500").getAsJsonObject();
+
+			    Set<Entry<String, JsonElement>> entrySet = f2.entrySet();
+			    for(Map.Entry<String,JsonElement> simOutputEle : entrySet){
+			    	if(simOutputEle.getValue().isJsonObject()){
+			    		JsonObject simOutputObj = simOutputEle.getValue().getAsJsonObject();
+			    		for (String prop : propsArray) {
+				    		if (simOutputObj.has(prop)) System.out.println(prop +  " : " + simOutputObj.get(prop));	
+						} 
+
+			    	}
+			        System.out.println("key >>> "+simOutputEle.getKey());
+			        System.out.println("val >>> "+simOutputEle.getValue());
+			    }
+
+			    if(f2.isJsonObject()){
+			        JsonObject f2Obj = f2.getAsJsonObject();
+			        JsonElement f3 = f2Obj.get("f3");
+			    }
+
+			}
+			
+		} catch (FileNotFoundException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}	
+	}
+	
+	
+	public static void main(String[] args) {
+		CompareResults tm = new CompareResults();
+		String path = "/Users/jsimpson/git/adms/GOSS-GridAPPS-D/gov.pnnl.goss.gridappsd/applications/python/sim_output_object.json";
+		SimulationOutput simOutProperties = tm.getOutputProperties(path);
+		tm.getProp(simOutProperties);
+		
+		
+		String sim_output = "/Users/jsimpson/git/adms/GOSS-GridAPPS-D/gov.pnnl.goss.gridappsd/applications/python/sim_output.json";
```

Comment:
Would be better to have arguments for these inputs rather than your path.  Or possibly just the home directory.